features/torque2/client/job-features.pan

```diff
@@ -38,28 +22,82 @@ function torque_size_to_gb = {
     };
     if (match(ARGV[0], 'tb')) {
         size = split('tb', ARGV[0]);
-        return(to_long(size[0]) * 1000);
+        return(to_long(size[0]) * 1099511627776);
```

Comment:
I am also in favor of adding the units suggested by @victor-mendoza ... Please do it!